index.js

```diff
@@ -118,14 +119,12 @@ Browserify.prototype.require = function (file, opts) {
     var expose = opts.expose;
     if (file === expose && /^[\.]/.test(expose)) {
         expose = '/' + relativePath(basedir, expose);
-        expose = expose.replace(/\\/g, '/');
```

Comment:
Couldn't you use your helper function from below here instead?

Reply:
I'm not sure what you mean. My changes basically extended the existing `relativePath` method to replace backslashes with forward slashes, which means that adding the `.replace(/\\/g, '/')` after a `relativePath` call is no longer necessary.

I don't quite remember, but I think the problem was that `.replace(/\\/g, '/')` was missing after some of the other `relativePath` calls and so the path behaviour was inconsistent on Windows.